service/create/service.go

```diff
@@ -876,6 +890,27 @@ func (s *Service) processCluster(cluster awstpr.CustomObject) error {
 
 	}
 
+	if key.HasClusterVersion(cluster) {
+		// Create NAT gateway.
+		var natGateway resources.ResourceWithID
+		natGateway = &awsresources.NatGateway{
+			Name:   key.ClusterID(cluster),
+			Subnet: publicSubnet,
+			// Dependencies.
```

Comment:
FYI we follow some ordering when it comes to configure dependencies and settings in other projects. Here I do not mind, but I will be more picky as soon as we rewrite the operator to use Cloudformation.

Reply:
Ack. Yes good to be stricter on this for the rewrite.